packages/ui/src/shared/constants/settings-types.ts

```diff
@@ -148,6 +150,14 @@ type QueryTrackerUserDefaultACOSettings = {
     [key in `qt-stage::${Stage}::queryTracker::defaultACO`]: string;
 };
 
+export type Cluster = string;
+export type QueryTrackerLastPath = {
+    [key in `local::${Cluster}::queryTracker::lastPath`]: string;
```

Comment:
Isn't it better to use more specific name `::lastSpytClique`/`::lastDiscoveryPath`/`::lastSpytDiscoveryPath`?